readme.adoc

```diff
@@ -29,38 +29,35 @@ Table of contents overview:
 Your feedback and contributions are most welcome! Instructions for
 building and contributing can be found in <<appendixbuild>>.
 
+=== What to expect?
+
+Bitcoin is a huge subject and we couldn't possibly cover all of its aspects here. Nevertheless, we
+hope to discuss the necessary features to get you started as well as to enable you to further explore it on your own.
+
+There are lots of people involved in Bitcoin; as some of them have opposing opinions,
+there are resources that express contradictory ideas. However, we always
+attempt to stick to the facts, where opinions do not matter.
+
+=== Who wrote this?
+
+The main authors of this book are Kalle Rosenbaum and Linnéa Rosenbaum. This work
+was commissioned and funded by Chaincode Labs, a development centre that runs educational
+programs for developers who want to https://learning.chaincode.com/[learn] about Bitcoin development.
 
 === How is this organized?
 
-The project is sectioned into chapters of different topics. Each
+The project is sectioned into chapters covering different topics. Each
 chapter will guide you through a number of links to articles or
-videos, that we recommend reading, and will briefly discuss each
-link. The material presented is written by individuals who have
+videos that we recommend reading, and will briefly discuss each
+link. The reported material was written by individuals who have
 studied Bitcoin development for a long time.
 
-The links refer to external resources on platforms we can't
+The links refer to external resources on platforms we cannot
 control. We have therefore saved the linked articles locally in this
 repository, along with info on where it was copied from, and when. The
 resources are collected in link:sources/sources.html[a separate
-document] (`sources/sources.adoc`) organized by the chapter they are
-linked from. The links found in the chapters refer to the original
-sources, but if you don't have an internet connection, the links
-appear dead, or the content seems severely changed, you can read the
+document] (`sources/sources.adoc`) and organized by the chapter they are
+linked from. The links found within chapters refer to the original
+sources, but if you don't have an internet connection, the links will
+appear dead or the content severely changed. We thus advise you to read the
```

Comment:
There's a logical error here, I think. What I'm trying to say is that if either

* you don't have an internet connection
* links appear dead
* content is severely changed

we advise you to read the content locally instead.

Reply:
I misunderstood the sentence, my bad!